packages/app-mobile/root.tsx

```diff
@@ -846,6 +847,7 @@ async function initialize(dispatch: Dispatch) {
 	if (Setting.value('env') === 'dev') {
 		if (Platform.OS !== 'web') {
 			await runRsaIntegrationTests();
+			await runCryptoIntegrationTests();
 		} else {
 			logger.info('Skipping RSA tests -- not supported on mobile.');
 		}
```

Comment:
These tests really can't run on web? In that case please update the log message to say "Skipping encryption test...."

Reply:
Updated.